CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+
+## Intro
+
+Early into the AY12/13 academic year, Prof Tay Yong Chiang organized a supper for Computer Science freshmen at Tembusu College. The bunch of seniors who were gathered there put together a document for NUS computing freshmen. This is that document.
+
+Feel free to create a pull request to edit or add to it, and share it with other freshmen you know.
+
+It is now currently maintained and updated by NUS Hackers.
+
+## What is this document?
+
+This document is a [living document](https://en.wikipedia.org/wiki/Living_document) that aims to provide valuable resources and guidance to incoming
+computer science freshmen.
+
+## The goal of this document
+
+The primary goal of this document is to create a repository of knowledge that will cover various aspects
+of a freshmen's computing journey. We hope that this document will serve to ease the transition of new students
+into the School of Computing (SoC).
+
+## Contributing to the document
+
+As much as possible, we want to adhere to the the overarching ideas in the
+[awesome manifesto](https://github.com/sindresorhus/awesome/blob/main/awesome.md).
+
+In other words:
+- It's curation, not collection. - We want to avoid the document from being too lengthy or diluted.
+- Research if the stuff you're including is actually *awesome*. - Only put stuff in that you or another contributor can **personally** recommend.
+- You should rather leave stuff out than include too much. - If you're unsure, ask someone else, or comment it in the PR.
+- Comment on why something is *awesome* (Not relevant for advice) - If you're recommending a book, or a lecture, tell us why you think it's great.
+- Pay attention to grammar. - Ensure your stuff is grammatically correct, typo-free and has no Markdown formatting errors.
+
+## I'm ready to contribute! How do I start?
+
+**Make sure your content follows the general principles in [Contributing to the document](#contributing-to-the-document)**
+
+### If you're giving general advice and tips to freshmen
+- [ ] Remember to follow the [Code of Conduct](CODE-OF-CONDUCT.md). We will not approve
+anything that goes against the Code of Conduct, or is submitted in bad faith.
+- [ ] Make sure there is no grammatical errors.
+- [ ] There is no specific topic or format you have to follow, but we recommend you to
+try and give advice on certain topics or areas that other contributors might
+have missed out on. (It doesn't just have to be about school life, anything from general
+programming tips, career advice, life advice that you live by and think would be helpful
+works!)
+
+### If you want to suggest a new section/list (e.g. favourite programming languages to learn, etc.)
+- [ ] Make sure that the section is relevant to the document. If you're not sure, feel free to 
+add an issue to ask us.
+- [ ] Make sure that the section is not already covered in the document.
+- [ ] Make sure there is no grammatical errors.
+- [ ] You should explain why you think the section is great.
```

Comment:
I feel like this falls under point number 1 about being relevant to the document, and since we already mentioned "it's curation, not collection" elsewhere, do you think we can remove this line?

Reply:
With this and another comment below, the thought process is this is a last check before they send a PR, think it would be good to remind them and reiterate certain points. I'll keep it for now, but I think if we think it's too repetitive still, we can remove it after